### PR TITLE
Create necessary models.

### PR DESCRIPTION
**Pull request by stefanbesovic**
Added User, Character, Vote and Film entity with Role, Planet and Gender enums.

### CODE REVIEW

File: src/main/java/com/axiomq/starwars/entities/Film.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String name;
+
+    @ManyToMany(mappedBy = "films", fetch = FetchType.LAZY)
+    private Set<Character> characters = new HashSet<>();
+}
```
Comment (branimir-vujicic) [inline src/main/java/com/axiomq/starwars/entities/Film.java:27]: add equals, hash code and toString

File: src/main/java/com/axiomq/starwars/entities/Character.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    @ManyToMany(fetch = FetchType.LAZY)
+    @JoinTable(
+            name = "character_films",
+            joinColumns = @JoinColumn(name = "character_id"),
+            inverseJoinColumns = @JoinColumn(name = "film_id")
+    )
+    private Set<Film> films = new HashSet<>();
+    private Planet planet;
+    private Integer votersCount;
+}
```
Comment (branimir-vujicic) [inline src/main/java/com/axiomq/starwars/entities/Character.java:37]: add equals, hash code and toString

File: src/main/java/com/axiomq/starwars/entities/Vote.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        if (!(o instanceof Vote)) return false;
+        Vote vote = (Vote) o;
+        return id.equals(vote.id) && value.equals(vote.value);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(id, value);
+    }
+}
```
Comment (branimir-vujicic) [inline src/main/java/com/axiomq/starwars/entities/Vote.java:41]: add equals, hash code and toString

File: src/main/java/com/axiomq/starwars/entities/Film.java (hunk 1 above)
Comment (stefanbesovic) [reply src/main/java/com/axiomq/starwars/entities/Film.java:27]: Done.

File: src/main/resources/application.properties [hunk 4]
```diff
@@ -8,3 +8,5 @@ spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.MySQL8Dialect
 spring.jpa.hibernate.ddl-auto = update
 spring.jpa.show-sql=true
 spring.jpa.properties.hibernate.format_sql=true
+spring.jpa.properties.hibernate.globally_quoted_identifiers=true
+
```
Comment (branimir-vujicic) [inline src/main/resources/application.properties:12]: test

**Review by branimir-vujicic — APPROVED** [review]
approved

**Pull request merged**